consensus/expected.go

```diff
@@ -265,32 +265,12 @@ func (c *Expected) RunStateTransition(ctx context.Context, ts types.TipSet, ance
 // validateMining checks validity of the block ticket, proof, and miner address.
 //    Returns an error if:
 //    	* any tipset's block was mined by an invalid miner address.
-//      * the block proof is invalid for the challenge
 //      * the block ticket is incorrectly computed
 //      * the block ticket fails the power check, i.e. is not a winning ticket
 //    Returns nil if all the above checks pass.
 // See https://github.com/filecoin-project/specs/blob/master/mining.md#chain-validation
 func (c *Expected) validateMining(ctx context.Context, st state.Tree, ts types.TipSet, parentTs types.TipSet) error {
 	for _, blk := range ts.ToSlice() {
-		parentHeight, err := parentTs.Height()
-		if err != nil {
-			return errors.Wrap(err, "failed to get parentHeight")
-		}
-
-		nullBlockCount := uint64(blk.Height) - parentHeight - 1
-		challengeSeed, err := CreateChallengeSeed(parentTs, nullBlockCount)
-		if err != nil {
-			return errors.Wrap(err, "couldn't create challengeSeed")
-		}
-
-		isValid, err := proofs.IsPoStValidWithVerifier(c.verifier, []proofs.CommR{}, challengeSeed, []uint64{}, blk.Proof)
```

Comment:
We aren't using PoSt when mining, so this whole chunk of code is defunkt.

Reply:
please have a TODO here in the code that we need to check some kind of vdf, and reference an issue filed against testnet to that effect

Reply:
This is now tracked in the "verification" bullet of #2119.

Reply:
Done.